src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/IdentifiersShouldNotMatchKeywordsMemberRuleTests.cs

```diff
@@ -601,15 +598,5 @@ End Sub
                 // Include the type parameter name but not the constraint.
                 GetBasicResultAt(3, 28, IdentifiersShouldNotMatchKeywordsAnalyzer.MemberRule, "C(Of T).for()", "for"));
         }
-
-        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
-        {
-            return new IdentifiersShouldNotMatchKeywordsAnalyzer();
-        }
-
-        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
-        {
-            return new IdentifiersShouldNotMatchKeywordsAnalyzer();
-        }
```

Comment:
Moved to the "common code" file. These methods are used by the tests for all four rules.